users-service/service.yml

```diff
@@ -25,14 +25,25 @@ messages:
     - user deleted
     - user not created
 
-
-dependencies:
+dependencies: # comment out this dependency for deployment
```

Comment:
Do we need to list the mongo dependencies in the services if its in the app dependencies?

Reply:
The dependency listed in application defines the config for mongo, and then this is saying "use the dependency configured in application.yml"

Reply:
Hmm, so this dependency isn't passed to every service similar to exocom?

Reply:
Correct, not every service needs it.

Reply:
I don't like that inconsistency, but okay sounds good.

Reply:
Yes, it's more of so that every service doesn't need to list the same configuration. We came up with this idea a while ago but we can refine it as we get feedback.